evalml/tests/component_tests/test_lsa.py

```diff
@@ -127,12 +96,12 @@ def test_int_col_names():
               'I dreamed a dream in days gone by, when hope was high and life worth living',
               'Red, the blood of angry men - black, the dark of ages past']
          })
-    lsa = LSA(text_columns=[4.75, -1])
+    lsa = LSA()
     lsa.fit(X)
     expected_col_names = set(['LSA(4.75)[0]',
                               'LSA(4.75)[1]',
-                              'LSA(-1)[0]',
-                              'LSA(-1)[1]'])
+                              'LSA(-1.0)[0]',
```

Comment:
Why did this column label become a float versus the original int?

Reply:
Good question! I believe it should have been a float to begin with:
![image](https://user-images.githubusercontent.com/5422235/103847390-1db5a400-506e-11eb-8efa-da8c05e35d54.png)

However, because we previously asked users to pass in text col names and then use those col names and we previously passed in -1, we got -1 as an int instead.